### ISSUE TITLE
Retrain EQT

### ISSUE THREAD

**Issue by Pengling2022** [seisbench/seisbench#284]
Hello,  @yetinam 
When I used the seisbench process to retrain EQTransformer, I encountered some problems and questions and wanted to ask you for help! Thank you so much!
1. I found in other questions that it was mentioned that there is filter and other preprocessing in the original EQT. Is there a corresponding part in seisbench? (I didn’t find annotate/classify)
2. For data that is not 60s, is this cropping random? (Possibly cropping out incomplete signal segments?)
3. There is an adjustment to class balance - class_weight in the original EQTransformer. Is it reflected in seisbench?
4. For eqtransformer_stead, has it been trained by seisbench? Or should we directly convert tensorflow's EQT model into a pytorch version model? If it is retrained, is the trained loss similar to the original version? Is there any evaluation comparison of F1 scores?

best wishes!

**Comment by yetinam** [comment]
Hi @Pengling2022 ,
thanks for your interest in SeisBench. I think all of your questions are answered in the implementation of our benchmark [here])(https://github.com/seisbench/pick-benchmark/) and the [associated paper](https://doi.org/10.1029/2021jb023499).   Feel free to reopen this issue if further questions remain.
